### ISSUE TITLE
IE9 and IE10 does not display CAPTCHA image sometimes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Daniel3356/recaptcha#170]
```
What steps will reproduce the problem?
1. Render the recaptcha widget with a PNG image in IE9 or IE10
2.
3.

What is the expected output? What do you see instead?
the CAPTCHA image should be displayed if its a jpeg or png.  If an image is PNG 
it does not get displayed.    

What version of the product are you using? On what operating system?
o.o.7
Windows Server 2003
Windows Server 2008


Please provide any additional information below.
Recaptcha image URLs never have a file extension (like .jpg or .png) so the 
browser must rely on the Content-Type header or else must sniff the image file 
to guess its file type. The Google server is sending the Content-Type header 
for CAPTCHA images, but it is always set to "image/jpeg". However, some of the 
images are really PNG images. This is problematic because the server is also 
sending the "x-content-type-options: nosniff" header. IE9 and IE10 honor this 
header and therefore do not display non-JPEG images. Other browsers apparently 
ignore this header because they display the PNG images fine.

The bottom line is that Content-Type header is wrong 30-40% of the time.
```

Original issue reported on code.google.com by `ds_consu...@securian.com` on 18 Jul 2013 at 8:13

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the very detailed bug report; these kinds of reports are excellent 
and really help us fix issues fast.  

Your analysis of the bug was spot on, and this bug has since been fixed.  There 
was a mixup that caused a small fraction of our png captchas to be mislabelled 
as jpegs.

~The ReCaptcha Team
```

Original comment by `cbra...@google.com` on 19 Jul 2013 at 2:31

**Comment by GoogleCodeExporter** [comment]
```
are there any isues with google recaptch feature?
we are getting unclear images and hard to login the sites.

Please let us know.
```

Original comment by `nracha...@gmail.com` on 10 Jan 2014 at 10:05

**Comment by GoogleCodeExporter** [comment]
```
even on this page image is not clear.it is started today.
```

Original comment by `nracha...@gmail.com` on 10 Jan 2014 at 10:33

**Comment by GoogleCodeExporter** [comment]
```
You said its fixed, but its not.
We are having that issue and seems i am not the only one.
```

Original comment by `carnava...@gmail.com` on 9 Jun 2014 at 8:58

**Comment by GoogleCodeExporter** [comment]
```
any solution yet, it is not solved yet
```

Original comment by `dotnetpe...@gmail.com` on 6 Aug 2014 at 6:58

**Comment by GoogleCodeExporter** [comment]
```
How can I force captcha to send jpg images only
```

Original comment by `dotnetpe...@gmail.com` on 6 Aug 2014 at 7:16

**Comment by GoogleCodeExporter** [comment]
```
The original poster had this to say:

"the server is also sending the "x-content-type-options: nosniff" header. IE9 
and IE10 honor this header and therefore do not display non-JPEG images. Other 
browsers apparently ignore this header because they display the PNG images 
fine. The bottom line is that Content-Type header is wrong 30-40% of the time."

As of February 18th 2015, this is still an issue - after an implementation of a 
recent project, we began getting bombarded with emails and screenshots of the 
reCAPTCHA images not loading. After some investigation, we discovered that 
there are PNG images supplied roughly 3-4 times out of 10 (the same results the 
OP quoted), and IE 9/10 will not render them. 

The issue was reported 18 months ago, and nothing has been done? I don't 
understand how  cbra...@google.com made the reply and didn't understand what 
the issue is. Mislabeled files aren't causing the content-type header to be 
incorrect.
```

Original comment by `jrbenall...@gmail.com` on 19 Feb 2015 at 3:04

**Comment by GoogleCodeExporter** [comment]
```
any solution, it is not solved yet?
```

Original comment by `lmtz...@gmail.com` on 4 Mar 2015 at 10:46

**Comment by GoogleCodeExporter** [comment]
```
Obscure reference... "anyone, anyone..." 
https://www.youtube.com/watch?v=uhiCFdWeQfA
```

Original comment by `smitma...@gmail.com` on 20 Mar 2015 at 12:31
